### ISSUE TITLE
Post with async-http-client

### ISSUE THREAD

**Issue by Tower450** [matt2xu/async-http-client#4]
C'est plus une question qu'une issue, est-ce que ton async-http-client support les POST? je regardez le code et j'en suis pas certain.

Merci de m'éclaircir!

**Comment by xenzh** [comment]
Hi, if you're asking whether this library supports POST, I'm pretty sure it does: https://github.com/matt2xu/async-http-client/blob/master/src/lib.rs#L141

**Comment by Tower450** [comment]
thanks bro! :)

**Comment by Tower450** [comment]
i am coding a CLI right now and i need some async HTTP request since rust 1.25 it supports async i think you guys are doing the first async client in Rust! keep going guys! Right now i am using Easy Curl which is doing the job but i think i will migrate to yours when it will be ready.

**Comment by xenzh** [comment]
This project is good as an example how to use `tokio` and futures, but for real projects you'd probably want to consider `hyper` or `reqwest` - they both are mature, well-maintained and do have async http clients included.

**Comment by Tower450** [comment]
yeah i tried both but i think your semantic is interesting, thanks a lot for your help
